src/components/Gallery.tsx

```diff
@@ -1,8 +1,11 @@
-import ImgixAPI, { APIError } from 'imgix-management-js';
 import { Component } from 'react';
-import Imgix from 'react-imgix';
-import { SourceProps } from './Dialog';
+import ImgixAPI, { APIError } from 'imgix-management-js';
 import { DialogExtensionSDK } from 'contentful-ui-extensions-sdk';
+
+import { SourceProps } from './Dialog';
+import { GalleryImage } from './GalleryImage/GalleryImage';
+import { GalleryImageSelectButton } from './/GalleryImageSelectButton/GalleryImageSelectButton';
+
```

Comment:
Changed import chunking for added clarity, so that it follows: external ->  internal -> css modules convention.